engine/src/main/java/com/agiletec/aps/system/services/lang/ISO_639 -1_langs.xml

```diff
@@ -553,6 +553,10 @@
 		<descr>Chinese</descr>
 	</Lang>
 	<Lang>
+		<code>zhs</code>
+		<descr>Chinese Simplified</descr>
+	</Lang>
```

Comment:
it is Simplified Chinese, not Chinese Simplified

we also need to amend the other one to Traditional Chinese